### ISSUE TITLE
Accept user action required action should have context key

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#273167]
The accept chat action keybinding intentionally does not have a strict `when` clause. However, this causes the notification action with the same kb to not work. We should set the weight of notification one to be higher.

```
{
  "key": "ctrl+shift+a",
  "command": "workbench.action.chat.focusConfirmation",
  "when": "accessibilityModeEnabled && chatIsEnabled"
}
```

cc @jooyoungseo, @rperez030
